scijava/scijava-ops/src/test/java/org/scijava/ops/hints/AdaptationHintTest.java

```diff
@@ -0,0 +1,72 @@
+package org.scijava.ops.hints;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.function.Function;
+
+import org.junit.Test;
+import org.scijava.ops.AbstractTestEnvironment;
+import org.scijava.ops.OpField;
+import org.scijava.ops.core.OpCollection;
+import org.scijava.ops.function.Computers;
+import org.scijava.ops.hints.BaseOpHints.Adaptation;
+import org.scijava.ops.hints.impl.DefaultHints;
+import org.scijava.ops.matcher.OpMatchingException;
+import org.scijava.plugin.Plugin;
+
+@Plugin(type = OpCollection.class)
+public class AdaptationHintTest extends AbstractTestEnvironment {
+
+	@OpField(names = "test.adaptation.hints")
+	public final Function<Double[], Double[]> op = (in) -> new Double[in.length];
+
+	@Test
+	public void testAdaptation() {
+		// make sure we can find the Op when adaptation is allowed
+		Hints hints = new DefaultHints();
+		hints.setHint(Adaptation.ALLOWED);
+		ops.env().setHints(hints);
+		@SuppressWarnings("unused")
+		Computers.Arity1<Double[], Double[]> adaptable = ops.op(

```

Comment:
@gselzer Could you add a `Hints` signature to the final methods of `OpBuilder` so that you can build an `Op` call with a particular set of `Hints`? (instead of requiring it to be set at the environment level)

Reply:
Done!